script.grab.fanart/.travis.yml

```diff
@@ -0,0 +1,20 @@
+dist: xenial
```

Comment:
Remove from submission

Reply:
Sorry about that, I thought the addon submitter addon would remove these (I think it does for my other addons). I'll get to work on it.